backend-shared/db/app_project_managed_env_test.go

```diff
@@ -66,4 +76,86 @@ var _ = Describe("AppProjectManagedEnvironment Test", func() {
 
 	})
 
+	Context("Test CountAppProjectManagedEnvironmentByClusterUserID function", func() {
+		It("should return 0 if there are no clusterusers", func() {
+			appProjEnv := &db.AppProjectManagedEnvironment{}
+			rows, err := dbq.CountAppProjectManagedEnvironmentByClusterUserID(ctx, appProjEnv)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(rows).To(BeZero())
+		})
+
+		It("should return the number of rows of AppProjectManagedEnvironment for a given clusteruser", func() {
+			By("add sample AppProjectManagedEnvironment with a given clusteruser")
+			expectedRowCount := 1
+			_, managedEnvironment, _, _, _, err := db.CreateSampleData(dbq)
```

Comment:
Is it possible to common up line 90 to 108 somehow? It is common in the next test case, and it looks similar to what is in the previous existing test too. (Although, the test case above to test  "return 0" doesn't require this, it might not be applicable to BeforeEach.

Reply:
I've moved it to a separate function and reused it when required.